src/Mod/Spreadsheet/Gui/DlgSettings.ui

```diff
@@ -20,6 +20,64 @@
    <string>Spreadsheet</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
+   <item row="1" column="0">
+    <widget class="QGroupBox" name="groupBox">
+     <property name="title">
+      <string>Display Settings</string>
+     </property>
+     <layout class="QGridLayout" name="gridLayout">
+      <item row="0" column="2">
+       <widget class="Gui::PrefLineEdit" name="formatString">
+        <property name="text">
+            <string notr="true">%V = %A</string>
+        </property>
+        <property name="toolTip">
+         <string>The format of the alias string presentation.
```

Comment:
Something more generic, like "The format of the custom cell string presentation" ?